packages/kv-store/src/Redis/Config/RedisConfig.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Tempest\KeyValue\Redis\Config;
+
+use UnitEnum;
+
+final class RedisConfig
```

Comment:
Shouldn't this implement `HasTag`?

Reply:
I actually don't think tag support for this feature is necessary, like emails. I'll remove it.